examples/odf/README.md

```diff
@@ -8,38 +8,31 @@ You can define the IBM Cloud credentials in the IBM provider block but it is rec
 
 Go [here](../../CREDENTIALS.md) for details.
 
-**NOTE**: These credentials are not required if running this Terraform code within an **IBM Cloud Schematics** workspace. They are automatically set from your account.
+You also need to install the [IBM Cloud cli](https://cloud.ibm.com/docs/cli?topic=cli-install-ibmcloud-cli) as well as the [OpenShift cli](https://cloud.ibm.com/docs/openshift?topic=openshift-openshift-cli)
+
+Make sure you have the latest updates for all IBM Cloud plugins by running `ibmcloud plugin update`.  
+
+**NOTE**: These requirements are not required if running this Terraform code within an **IBM Cloud Schematics** workspace. They are automatically set from your account.
 
 ## 2. Test
 
 ### Using Terraform client
 
 Follow these instructions to test the Terraform Module manually
 
-Create the file `test.auto.tfvars` with the following input variables, these values are fake examples:
+Create a file `terraform.tfvars` with the following input variables:
 
 ```hcl
-enable                  = true
+is_enable               = true
 ibmcloud_api_key        = "<api-key>"
-
-// Cluster parameters
-kube_config_path        = ".kube/config"
-worker_nodes            = 2  // Number of workers
-
-// ODF parameters
-resource_group_name     = "default"
-region                  = "us-east"
-cluster_id              = "<cluster-id>"
+cluster                 = "<cluster-id>"
```

Comment:
Do you need to list examples of the additional parameters?

Reply:
added